R4ML/R/zzz.R

```diff
@@ -85,286 +71,128 @@ auto_start_session <- function() {
 # call the systemML as well as SparkR functionality as needed
 # if sparkR is already initialized then it wil re-initialize it
 .onLoad <- function(libname, pkgname) {
-  logsrc <- r4ml.env$PACKAGE_NAME
-
-  if ("SparkR" %in% loadedNamespaces()) {
```

Comment:
We should let R handle loading required packages from the DESCRIPTION. This is a CRAN requirement.